cookbook/integration_openai_sdk.ipynb

```diff
@@ -14,14 +14,18 @@
     },
```

Comment:
<p>we do not talk about <code>users</code> </p><p> </p><p>"Optionally, you can pass your own trace_id to langfuse if you want to sync it with your internal ids. Just pass it as a <code>langfuse_observation_id</code> kwarg. </p><p> </p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/langfuse/langfuse-docs/pull/415/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>